src/interfaces/WF/WFTimeUntilReferenceDate.ts

```diff
@@ -0,0 +1,6 @@
+type WFTimeUntilReferenceDate = (
+  'Right Now'
+  | 'Other'
+  );
```

Comment:
Could the leading spaces be removed from the start of this line please?

Reply:
Fixed